src/App.tsx

```diff
@@ -1,7 +1,31 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: string
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    key: 'Nothing was pressed yet',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleClick);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleClick);
+  }
+
+  handleClick = (event: KeyboardEvent) => {
+    this.setState({ key: `The last pressed key is [${event.key}]` });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <p className="App__message">{this.state.key}</p>
```

Comment:
and here you can have
{this.state.key ? \`The last pressed key is [${this.state.key}]\` : 'Nothing was pressed yet'}